src/main/java/org/duckdb/DuckDBDriver.java

```diff
@@ -20,6 +21,8 @@ public class DuckDBDriver implements java.sql.Driver {
     static final String DUCKDB_URL_PREFIX = "jdbc:duckdb:";
     static final String MEMORY_DB = ":memory:";
 
+    static final ScheduledThreadPoolExecutor scheduler = new ScheduledThreadPoolExecutor(1);
```

Comment:
It would be helpful when the thread name has a meaningful name. We could achieve this by using a `ThreadFactory` as second argument to set the name a new thread is created.

Reply:
Thanks for the review! Good idea, updated the PR to add this.